ale/drivers/messenger_drivers.py

```diff
@@ -164,3 +182,59 @@ def starting_ephemeris_time(self):
             sclock = self.label['IsisCube']['Archive']['SpacecraftClockStartCount']
             self._starting_ephemeris_time = spice.scs2e(self.spacecraft_id, sclock)
         return self._starting_ephemeris_time
+
+    @property
+    def _focal_length(self):
```

Comment:
Why is the focal length private?

Reply:
I quite literally rearranged code that already existed, so I did not think to check signatures...

Reply:
_focal_plane_temperature above is also private, but it is only used within focal_length. Do I need to make this public for testing purposes, or is it fine to simply test focal_length?

Reply:
you can test private things in Python, the `_` for private is only a suggestion.